### PR TITLE
Integration tests for crowdsourcing API

### PR DESCRIPTION
**Pull request by ravsa**
Integration tests for crowdsourcing API endpoints:

- get master tag list for given ecosystem

- get next component to be tagged for given ecosystem

- set tags to component

### CODE REVIEW

File: integration-tests/features/steps/user_tag.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+@then("I should get 401 status code as response for next untagged component")
+def check_status_code_for_get_next_untagged_component(context):
+    """Check get next untagged component require authorization tokens."""
+    assert context.response.status_code == 401
+
+
+@then("I should get a 200 status code and component as {response} type")
+def check_get_next_untagged_component_response(context, response):
+    """Check that response contains the component."""
+    data = context.response.json()
```
Comment (tisnik) [inline integration-tests/features/steps/user_tag.py]: I'd swap the line #106 and #107 + assert if the status code is really 200 (it does not make sense to test JSON content if code != 200)

File: integration-tests/features/steps/user_tag.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    """Check master tag list require authorization tokens."""
+    assert context.response.status_code == 401
+
+
+@then('I should get json object contains tag_list which is a array of strings')
+def check_master_tag_list_response_json(context):
+    """Check that json response contains master tags."""
+    json_data = context.response.json()
+    assert 'ecosystem' in json_data
+    assert 'tag_list' is not any(json_data.get('tag_list', []))
```
Comment (tisnik) [inline integration-tests/features/steps/user_tag.py]: I'm not sure what's checked here. Is it just a test for non empty list?

Comment (tisnik) [reply integration-tests/features/steps/user_tag.py]: typo: an array

Comment (ravsa) [reply integration-tests/features/steps/user_tag.py]: yes, It's a test for a non-empty tag list.

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
LGTM. Thanks a lot!

**Pull request merged**